docs/components/control/authenticate-with-callback.mdx

```diff
@@ -10,11 +10,12 @@ import { Tab, Tabs } from 'nextra-theme-docs';
 
 ---
 
-The `<AuthenticateWithRedirectCallback/>` is used to complete a custom OAuth flow. Simply render the component under the route you passed as `redirectUrl` to the `authenticateWithRedirect` methods.
+The `<AuthenticateWithRedirectCallback />` is used to complete a custom OAuth flow. Simply render the component under the route you passed as `redirectUrl` to the `authenticateWithRedirect` methods.
 
 ## Usage
 
-<Tabs items={['Next.js', 'React', 'Remix', 'Gatsby']}>
+/* TO-DO: Try to use CodeBlockTabs when they are ready */
```

Comment:
They should be ready, did you run into an issue?

Reply:
I think I read in the contributing file that if the code has content other than code (like text, code, then more text), then use tabs. If it's just switching between code content, use codeblocktabs. I forgot to go through these and apply that logic !!